static_src/routes.js

```diff
@@ -57,10 +51,6 @@ export function overview(next) {
     })
     .then(pageActions.loadSuccess, pageActions.loadError);
 
-  ReactDOM.render(<MainContainer>
```

Comment:
No, I had originally moved it inside the callback, which obviously blocked rendering, and when I moved it out stuck it above the `spaceActions` fetch call for no good reason 😄